### ISSUE TITLE
NVDA cannot read Windows Terminal (cmd.exe) when run as administrator

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#20368]
### Steps to reproduce
* Press `windows+R`
* Type `cmd.exe`
* press `control+shift+enter`; this runs the command as administrator
* if windows request you identifying information as administrator or UAC, enter the requested information and validate
* try to read the terminal's content with review cursor
* try to type in the terminal and enter commands

### Actual behavior
NVDA does not report anything. Errors are logged.

### Expected behavior
NVDA should report the content of the terminal and its modifications.
If it's not possible but if there is a workaround, it would be useful mentioning it in the User Guide's dedicated section.
### NVDA logs, crash dumps and other attachments
```
ERROR - scriptHandler.executeScript (11:18:42.329) - MainThread (12980):
error executing script: <bound method GlobalCommands.script_review_nextLine of <globalCommands.GlobalCommands object at 0x0000023E87BBC440>> with gesture 'pavNum 9'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 300, in executeScript
  File "globalCommands.pyc", line 1972, in script_review_nextLine
  File "speech\speech.pyc", line 1555, in speakTextInfo
  File "speech\types.pyc", line 47, in __iter__
  File "speech\speech.pyc", line 1595, in getTextInfoSpeech
  File "winConsoleHandler.pyc", line 283, in getTextWithFields
  File "wincon.pyc", line 116, in ReadConsoleOutput
PermissionError: [WinError 5] Accès refusé.
```
#### NVDA installed/portable/running from source
Installed
#### NVDA version
2026.2beta3
#### Windows version
Windows 11 23H2 (AMD64) build 22631.6783
#### Other information about your system
N/A
#### Does the issue still occur after restarting your computer?
Not yet tested
#### Have you tried any other versions of NVDA? If so, please report their behaviors
Tried 2025.3.3 portable with no more luck. However such restriction is probably expected on portable versions.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the System Accessibility Repair Tool in NVDA's tools menu?
Not tested

**Comment by Emil-18** [comment]
@CyrilleB79 I tested the classic terminal, both with UIA enabeled and dissabeled with NVDA 2026.1.1. I had no problems. Have you tried to run NVDA explisitly as administrator? I have heard about someone that have a problem that causes NVDA to not run as admin by default

**Comment by thgcode** [comment]
I found the following documentation about uiAccess="true" on the manifest https://v3.wails.io/guides/windows-uac/ .
It seems that to read the elevated Windows command prompt, the program needs to be run from an administrator-only controlled location (Program files for example), declare the UIAccess property on its manifest, and be signed. If NVDA is moved to another location or it is not signed, according to this documentation, Windows will not grant access to read the elevated command prompt.

**Comment by seanbudd** [comment]
@CyrilleB79 - can we get a full log to confirm this is happening on an installed copy?

Is this happening with "Over the shoulder" style administration? i.e. is the UAC prompt for running as a different administrator account, or elevating your current account as admin? if you are running the terminal from a separate account, which is administrator, we can't read it and this is a known limitation.

**Comment by seanbudd** [comment]
Can you also provide `nvda+f1`?

**Comment by CyrilleB79** [comment]
> if you are running the terminal from a separate account, which is administrator, we can't read it and this is a known limitation.

Yes, I'm exactly in this case. I didn't know that.

Maybe we can document it in the UG?

Also, where does this limitation come from? Should we ask Microsoft about it? Is Narrator impacted the same way?

**Comment by SaschaCowley** [comment]
We're happy to document this in the user guide.
This is a Windows security feature to stop user accounts interfering with each other. Your administrator terminal is running in a different account. Enabling NVDA to interact with that process could allow it to perform actions as that account (so your user NVDA would essentially become administrator).
